README.md

```diff
@@ -145,9 +144,16 @@ npm install --save jupyterlab
 git clone https://github.com/jupyter/jupyterlab.git
 cd jupyterlab
 npm install
-npm run build:all
+npm run build
+cd jupyterlab  # into the server extension
+npm install
+npm run build
 ```
 
+Note: If using an earlier version of node (<4), it is advised that you
+run `npm dedupe` after `npm install`, since older versions of node
+did not install dependencies maximally flat, which can cause version conflicts.
```

Comment:
I use "did" implying they were released in the past, but I could go either way here.